src/sensors/gps_display.cpp

```diff
@@ -178,9 +178,14 @@ void GPSDisplay::paint(QPainter* painter)
 		return;
 	QPointF gps_pos = widget->mapToViewport(gps_coord);
 	
+	// Highlight markers by underlaying white under the tracking color
+	highlighting += 64;
+	if (highlighting > 2 * 192)
+		highlighting = 0;
+	auto underlay = QColor{255, 255, 255, 63 + (qAbs( 192 - highlighting))};
```

Comment:
The four-`int` `QColor` construction is somewhat difficult to read. How about `QColor(Qt::white).setAlpha(63 + qAbs(192 - highlighting))`?

I also don't see readily why `highlight` increases by 64 and what is the magic 192. IMHO there is too much complicated code around simple periodic alpha change. Good illustration is that I overlooked the addition of 63 in alpha value above. The parentheses around `qAbs()` are extra.